apps/hmc/hmc-hmi-outbound-adapter/perftest.yaml

```diff
@@ -5,6 +5,17 @@ metadata:
 spec:
   values:
     java:
+      replicas: 2
+      autoscaling:
+        enabled: true
+        maxReplicas: 4
+      memoryRequests: '1024Mi'
+      memoryLimits: '1536Mi'
+      cpuRequests: '250m'
```

Comment:
Same story with memory requests, but for this also CPU should be at least 300m. If it is known to be fairly stable, avg utilization could be increased from the default 80%.